packages/web-components/src/components/ic-toast/ic-toast.tsx

```diff
@@ -0,0 +1,318 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Method,
+  Prop,
+  State,
+} from "@stencil/core";
+import closeIcon from "../../assets/close-icon.svg";
+import errorIcon from "../../assets/error-icon.svg";
+import infoIcon from "../../assets/info-icon.svg";
+import neutralIcon from "../../assets/neutral-icon.svg";
+import successIcon from "../../assets/success-icon.svg";
+import warningIcon from "../../assets/warning-icon.svg";
+import { getSlot, isSlotUsed } from "../../utils/helpers";
+import {
+  IcActivationTypes,
+  IcStatusVariants,
+  IcThemeForegroundEnum,
+} from "../../utils/types";
+import { ActionAreaElementTypes } from "./ic-toast.types";
+
+const ICONS = {
+  neutral: {
+    icon: neutralIcon,
+    alt: "Neutral",
```

Comment:
Should these be called `ariaLabel`? When I saw it as alt, I assumed it was alternative to the icon?